tools/oss-fuzz.sh

```diff
@@ -32,7 +32,7 @@ meson_args=("-Db_lundef=false")
 if [ -z "$FUZZING_ENGINE" ]; then
     meson_args+=("-Dllvm-fuzz=true")
 else
-    meson_args+=("-Doss-fuzz=true" "--auto-features=disabled" "-Dnspawn=enabled" "-Dresolve=true")
+    meson_args+=("-Doss-fuzz=true" "--auto-features=disabled" "-Dremote=auto" "-Dnspawn=enabled" "-Dresolve=true")
```

Comment:
Extra dot in commit title.

I'd make the comment about too-long lines here too.